index.bs

```diff
@@ -92,13 +101,16 @@ spec:dom-ls; type:interface; text:Document
     the visibility and position of DOM elements relative to a root element or
     the top level document's viewport.
 
-<h3 id='2-1-intersection-observer-callback'>The IntersectionObserverCallback</h3>
-<pre class="idl">
-  callback IntersectionObserverCallback = void (sequence&lt;IntersectionObserverEntry> entries, IntersectionObserver observer)
-</pre>
-This callback will be invoked when there are changes to <i>target</i>'s intersection with <i>root</i>, as per the <a>processing model</a>.
+<h3 id='2-1-intersection-observer-callback'>
+The IntersectionObserverCallback</h3>
+    <pre class="idl">
+        callback IntersectionObserverCallback = void (sequence&lt;IntersectionObserverEntry> entries, IntersectionObserver observer)
+    </pre>
+    This callback will be invoked when there are changes to <i>target</i>'s
+    intersection with <i>root</i>, as per the <a>processing model</a>.
 
-<h3 id='2-2-intersection-observer-interface'>The IntersectionObserver interface</h3>
+<h3 id='2-2-intersection-observer-interface'>
+The IntersectionObserver interface</h3>
```

Comment:
Oh, or just _this_, which I see you've used. (There is no good agreement on how to refer to "this", so it seems fine to say _this_ since it's clear.)